defactor-com/ui-kit
# Feat/add pie and bar charts 73 74

## Add Bar Chart, Pie Chart and refactor Dashboard component to customize the content

### What does this PR do?

- Resolve #73
- Resolve #74
- Refactor Dashboard to receive the main content and the items from the right column as params
- Add simple tooltip to the Bar and Pie Chart
- Update to 0.0.46 version

### Steps to test

1. Run the project locally
2. Go to Dashboard component
3. Check the three examples

### Screenshots

**Stories**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/dff960b7-e806-48bc-85b2-8309204886e3)

**Desktop breakpoint**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/5b11d16b-0637-46ff-8202-007ebf3e0799)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/865b46de-b1d4-4e62-b591-557c9b06af4b)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/78c35de9-5871-45b7-ae8a-70463b58a88d)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/54c15999-61bc-46a2-8563-91e5c198a94c)

**Mobile breakpoint**

![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/9925ffcf-5657-4891-ba14-53091e5b6d94)
![imagen](https://github.com/defactor-com/ui-kit/assets/66583677/b37f5bff-1013-4760-8ecd-f50dc537f315)
